packages/analytics/analytics-utilities/src/types/explore/requests.ts

```diff
@@ -28,18 +28,78 @@ export const queryableRequestDimensions = [
   'service_port',
   'service_protocol',
   'sse',
+  'status_code',
   'status_code_grouped',
   'time',
   'upstream_uri',
+  'upstream_status_code',
   'upstream_status_code_grouped',
   'websocket',
 ] as const
 
 export type QueryableRequestDimensions = typeof queryableRequestDimensions[number]
 
-export const filterableRequestDimensions = makeFilterable(queryableRequestDimensions)
+// These are dimensions that can be fetched/searched via KSearch/KAnalytics/KHCP
+export const fetchableRequestDimensions = [
+  'api_product',
+  'api_product_version',
+  'application',
+  'consumer',
+  'consumer_group',
+  'control_plane',
+  'control_plane_group',
+  'data_plane_node',
+  'gateway_service',
+  'route',
+] as const satisfies QueryableRequestDimensions[]
+
+export type FetchableRequestDimensions = typeof fetchableRequestDimensions[number]
+
+// These are dimensions that can be searched locally (i.e. in-memory) in the frontend
+// Will have a pre-populated list that can be searched from defined in the host app
+export const searchableLocalRequestDimensions = [
```

Comment:
I think we will include `response_source` in this list

Reply:
Are we sure that it's a searchable one, not a text based one, is it something that will have a pre-populated list to select from?

Reply:
Yea, we would like to show "kong" and "upstream". It's in the first sheet here https://docs.google.com/spreadsheets/d/1XQW4sXPlR7_7kUKxhKGTNueeQUZF3Pr4FJnbRz00axE/edit?gid=0#gid=0